docs/install.md

```diff
@@ -17,6 +17,71 @@
 - [PMEM-CSI Deployment CRD](#pmem-csi-deployment-crd)
 - [Filing issues and contributing](#filing-issues-and-contributing)
 
+## Overview
+
+This section summarizes the steps that may be needed during the entire
+usage of PMEM in a cluster, starting with the initial preparations and
+ending with decommissioning the hardware. The other sections explain
+each step in more detail.
+
+When setting up a cluster, the administrator must install the PMEM
+hardware on nodes and configure some or all of that PMEM for usage by
+PMEM-CSI ([prerequisites](#prerequisites)]. Nodes where PMEM-CSI is
+supposed to run must have a certain [label in
+Kubernetes](#installation-and-setup).
+
+The administrator must install PMEM-CSI, using either [the PMEM-CSI
+operator](#install-using-the-operator) (recommended) or with [scripts
+and YAML files in the source code](#install-from-source). A PMEM-CSI
+installation can only use either [direct device
+mode](/docs/design.md#direct-device-mode) or [LVM
+device mode](/docs/design.md#direct-device-mode). It is possible to install
+PMEM-CSI twice on the same cluster with different modes, but then:
+
+- The driver names must be different.
+- The installations must run on different nodes by using different node
+  labels, or the "usage" parameter of the LVM mode driver installation
+  one a node must be so that it leaves spaces available for the direct mode
+  driver installation on that same node.
```

Comment:
Would it make sense to mention about providing the different `StorageClasses` for those drivers?

Reply:
Ignore this, I figured out that the next paragraph is having this.